dev/archery/archery/cli.py

```diff
@@ -353,7 +353,7 @@ def check_language(ctx, param, value):
                      is_flag=True,
                      help="Preserve workspace for investigation."),
         click.option("--output", metavar="<output>",
-                     type=click.File("w", encoding="utf8"), default="-",
```

Comment:
New to me.

> Files can be opened for reading or writing.  The special value ``-`` indicates stdin or stdout depending on the mode.

-- https://github.com/pallets/click/blob/0c85d80b07742b9f9ed4d583d694c37fce1d1292/src/click/types.py#L635